airflow/decorators/base.py

```diff
@@ -151,6 +151,7 @@ def _find_id_suffixes(dag: DAG) -> Iterator[int]:
     core = re.split(r"__\d+$", task_id)[0]
     return f"{core}__{max(_find_id_suffixes(dag)) + 1}"
 
+RV = TypeVar("RV", bound=Any)
```

Comment:
What does `bound=Any` do? Isn’t bounding the TypeVar to Any the same as not bounding?